src/test/java/io/vertx/openapi/test/E2ETest.java

```diff
@@ -86,6 +88,38 @@ void testExtractPath(String basePath, VertxTestContext testContext) {
     }).onFailure(testContext::failNow);
   }
 
+  //These types come from: https://spec.openapis.org/oas/latest.html#special-considerations-for-multipart-content
+  // 1 primitive "petId"
+  // 1 complex "json"
+  // 1 primitive with content encoding aka png base64 encoded.
+  @Test
+  @Timeout(value = 2, timeUnit = TimeUnit.SECONDS)
+  @DisplayName("Send a multipart/form-data request")
+  public void sendMultipartFormDataRequest(VertxTestContext testContext) {
+    Path path = getRelatedTestResourcePath(E2ETest.class).resolve("multipart.txt");
+
+
+    setupContract("", testContext).compose(v -> createValidationHandler(req -> {
+      testContext.verify(() -> {
+        assertThat(req.getBody()).isNotNull();
+        JsonObject jsonReq = req.getBody().getJsonObject();
+        assertThat(jsonReq.getLong("petId")).isEqualTo(1234L);
+        assertThat(jsonReq.getJsonObject("petMetadata").toString())
+          .isEqualTo("{\"friends\":[123,456,789],\"contactInformation\":{\"name\":\"Example\",\"email\":\"example@example.com\",\"phone\":\"5555555555\"}}");
```

Comment:
Please build a JsonObject and compare those JsonObject, because there is no order in JsonObject. So a String comparison may fail.

Reply:
I've updated it to compare against another JsonObject.